keanu-project/src/main/java/io/improbable/keanu/tensor/generic/GenericTensor.java

```diff
@@ -19,14 +19,22 @@
     private long[] shape;
     private long[] stride;
 
-    public static <T> GenericTensor<T> create(T data, long[] shape) {
+    public static <T> GenericTensor<T> createFilled(T data, long[] shape) {
         return new GenericTensor<>(shape, data);
     }
 
+    public static <T> GenericTensor<T> create(T[] data, long[] shape) {
+        return new GenericTensor<>(data, shape);
+    }
+
     public static <T> GenericTensor<T> scalar(T data) {
         return new GenericTensor<>(data);
     }
 
+    public static <T> GenericTensor<T> placeHolder(long[] shape) {
```

Comment:
@gordoncaleb is removing the `placeHolder` methods in another PR - it looks like you've already merged his changes - so it feels like the wrong design choice to add it back in here.